torch/_dynamo/backends/cudagraphs.py

```diff
@@ -44,6 +44,13 @@ def meta_fk(meta):
         elif n.op == "call_function":
             if n.target is operator.getitem:
                 continue
+            from torch._ops import HigherOrderOperator
```

Comment:
@eellison can we get your advice on this issue? Presumably there is a way to grab the schema of a higher order op after speculate_subgraph() has been run, but we're not all that familiar with it torch dynamo.

You can reproduce the issue by commenting out 

```
            if isinstance(n.target, HigherOrderOperator):
                continue
```

And running any of the newly added unit tests in this PR.

Reply:
@BoyuanFeng since you worked on https://github.com/pytorch/pytorch/commit/8bf0be7c78e59127bb3c9d9c75a51cbdd21efa8c, can you give your opinion on whether simply skipping over a higher order op is appropriate to do when checking for input mutations? My impression is no.

Reply:
> These ops provide schema.

They don't have schema currently so we need to support them case by case.

Reply:
cc @zou3519 on wanting to look at effects/schema of higher order op